quipucords/quipucords/celery.py

```diff
@@ -21,6 +23,13 @@
 task_packages = ["scanner", "scanner.satellite.five", "scanner.satellite.six"]
 app.autodiscover_tasks(task_packages)
 
+
+@signals.worker_init.connect
+def worker_init_hook(**kwargs):
+    """Run custom functions on worker initialization."""
+    start_debugger_if_required()
```

Comment:
How does this behave when there are `> 1` workers? Will they conflict over the same debugger port? Have you confirmed manually that this works?

Reply:
Excellent question! Before choosing this signal I was looking for something that would run only once, before starting all the worker threads. Turns out that `worker_init` is kinda misleading and does not execute for each worker (or I misunderstand the concept - isn't it the same as  `-c` argument, which defaults to number of CPUs detected?), so there's not conflict: only one instance of the debugger spawns here.